internal/core/items.rs

```diff
@@ -166,9 +167,11 @@ impl ItemRc {
         // lifetime of the component, which is 'a.  Pin::as_ref removes the lifetime, but we can just put it back.
         unsafe { core::mem::transmute::<Pin<ItemRef<'_>>, Pin<ItemRef<'a>>>(result) }
     }
+
     pub fn downgrade(&self) -> ItemWeak {
         ItemWeak { component: VRc::downgrade(&self.component), index: self.index }
     }
+
     /// Return the parent Item in the item tree.
     /// This is weak because it can be null if there is no parent
     pub fn parent_item(&self) -> ItemWeak {
```

Comment:
In retrospcect, this probably should have been 
```suggestion
    pub fn parent_item(&self) -> Option<Self> {
```

No need to abuse the Weak for that.